### PR TITLE
run: provide a hint if we get output duplication error

### PR DESCRIPTION
**Pull request by efiop**
Related to #2559

* [x] ❗ Have you followed the guidelines in the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) list?

* [x] 📖 Check this box if this PR **does not** require [documentation](https://dvc.org/doc) updates, or if it does **and** you have created a separate PR in [dvc.org](https://github.com/iterative/dvc.org) with such updates (or at least opened an issue about it in that repo). Please link below to your PR (or issue) in the [dvc.org](https://github.com/iterative/dvc.org) repo.

* [x] ❌ Have you checked DeepSource, CodeClimate, and other sanity checks below? We consider their findings recommendatory and don't expect everything to be addressed. Please review them carefully and fix those that actually improve code or fix bugs.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: dvc/exceptions.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def __init__(self, output, stages):
         )
         msg = (
             "file/directory '{}' is specified as an output in more than one "
-            "stage: {}"
+            "DVC-file, which is not allowed: {}\n"
```
Comment (jorgeorpinel) [inline dvc/exceptions.py]: Hmmm. We use the term "stage" in the console output a lot so I see no problem with just:

```suggestion
            "stage: {}\n"
```

The "which is not allowed" part is a little confusing in this sentence, maybe in the next line?

File: dvc/exceptions.py [hunk 2]
```diff
@@ -47,7 +47,8 @@ def __init__(self, output, stages):
...
+            "Consider using a different output name."
```
Comment (jorgeorpinel) [inline dvc/exceptions.py]:
```suggestion
            "This is not allowed. Consider using a different output name."
```

File: dvc/exceptions.py [hunk 3]
```diff
@@ -47,7 +47,8 @@ def __init__(self, output, stages):
...
         ).format(output, "\n    ".join(stages))
```
Comment (jorgeorpinel) [inline dvc/exceptions.py]: Maybe use back ticks with something like

```suggestion
         ).format(output, '`' + "`\n,  `".join(stages) + '`')
```

? (Probably not that exact code.)

File: dvc/exceptions.py (hunk 1 above)
Comment (jorgeorpinel) [reply dvc/exceptions.py]: See https://github.com/iterative/dvc/pull/2941#discussion_r356996806.

File: dvc/exceptions.py (hunk 3 above)
Comment (Suor) [reply dvc/exceptions.py]: We use single ticks inline and no quoting on separate lines everywhere else, so I would leave it as is.

Comment (jorgeorpinel) [reply dvc/exceptions.py]: True

**Pull request merged**